src/odemis/util/__init__.py

```diff
@@ -721,3 +721,39 @@ def bindFuture(future, fn, args=(), kwargs=None):
             future.set_exception(e)
     else:
         future.set_result(result)
+
+
+def sum_within_circle(data, i, j, radius):
```

Comment:
isn't there a better place for this function than src/odemis/util/__init__.py? Maybe src/odemis/util/spot.py